dist/bundle.js

```diff
@@ -0,0 +1,15 @@
+/*!
+ * The buffer module from node.js, for the browser.
```

Comment:
Why are we versioning the build output? Are we trying to support a browser, or Node?

Reply:
Added `target: 'node'` to `webpack.config.js` to prevent buffer from getting bundled.